### ISSUE TITLE
Readme is unclear

### ISSUE THREAD

**Issue by sparkrevolutions** [twilson63/express-couchUser#40]
Hi there. Firstly - cool module, crazy that this hadn't already been done well.

Im trying to use some of the basic methods by following the readme, but so far im having no luck.

Every request I make is denied (500, 403), essentially saying I don't have access to the user's table. I have tried a signin method with my DB admin credentials and then a sign up request - didn't work.

The readme and its examples don't seem to make any any reference to where admin credentials go, so couchUser can execute the views from the init file on my _users database. There is two different usage examples at the top and bottom of the readme, and I have no idea what the bottom one refers to:

Top
`var couchUser = require('express-couchUser');`
`var express = require('express');`
`var app = express();`

`// Required for session storage`
`app.use(express.cookieParser());`
`app.use(express.session({ secret: 'Use something else here.'}));`

`app.configure(function() {`
 ` app.use(couchUser({`
`    users: 'http://localhost:5984/_users',`
`    email: {`
`      ...`
`    },`
`    adminRoles: [ 'admin' ],`
`    validateUser: function(data, cb) {...}`
`  }));`
`});`

Bottom
`var user = require('express-user-couchdb');`
`var config = { `
`  users: 'http://localhost:5984/_users', `
`  email ....`
`};`
`app.config(function() {`
`  // handle other requests first`
`  app.use(express.router);`
`  // handle core requests`
  `app.use(user(config));`
`});`
`node init.js [couchdb users db]`

So in simple terms, what is the minimum requirement to get this module up and running. Have I missed anything obvious?

Cheers

**Comment by brentcobb** [comment]
Closing this issue because I just updated the readme.
